ui/widgets/spinner.js

```diff
@@ -225,12 +225,8 @@ $.widget( "ui.spinner", {
 
 				// Add buttons
 				.append(
-					"<a>" +
-						"<span>&#9650;</span>" +
-					"</a>" +
-					"<a>" +
-						"<span>&#9660;</span>" +
-					"</a>"
+					"<a></a>" +
+					"<a></a>"
```

Comment:
Can we just put this on one line? The linebreaks are pointless by now. Same for the one in `_buttonHtml()` below.